src/components/shared/Project/VolumeStatLine/VolumeStatLine.tsx

```diff
@@ -0,0 +1,70 @@
+import { Trans } from '@lingui/macro'
+import ETHAmount from 'components/shared/currency/ETHAmount'
+import CurrencySymbol from 'components/shared/CurrencySymbol'
+
+import { ThemeContext } from 'contexts/themeContext'
+import { Property } from 'csstype'
+import { BigNumber } from 'ethers/lib/ethers'
+import { useCurrencyConverter } from 'hooks/v1/CurrencyConverter'
+import { useContext, useMemo } from 'react'
+import { formatWad } from 'utils/formatNumber'
+
+import { textPrimary, textSecondary } from 'constants/styles/text'
+import { CurrencyName } from 'constants/currency'
+import StatLine from '../StatLine'
+
+export const VolumeStatLine = ({
+  totalVolume,
+  color,
+  convertTo,
+}: {
+  totalVolume: BigNumber | undefined
+  color: Property.Color
+  convertTo?: { currency: CurrencyName }
```

Comment:
**suggestion (non-blocking)**: thoughts on a `CurrencyName` prop instead for simplicity?

```suggestion
  convertToCurrency?: CurrencyName
```

Reply:
done! 🙌